frontend/src/components/ViewActivities/index.js

```diff
@@ -92,21 +97,26 @@ class ViewActivities extends React.Component {
       if (!this.state.addingActivity) {
         return (
           <div>
+            <Button type='button' onClick={this.addActivity}>+ Add</Button>
```

Comment:
Nit: Is "+ Add" redundant? If not then no change but if you think so consider removing the "+" or "Add".